README.md

```diff
@@ -64,6 +64,11 @@ Then perform the following commands on the root folder:
 - [Terraform](https://www.terraform.io/downloads.html) 0.11.x
 - [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v2.1
 
+### App Engine
+Note that this module requires a App Engine being configured in the specified project/region. 
+This is because Google Cloud Scheduler is dependent on the project being configured with App Engine.
+Documentation on the App Engine dependency is here: https://cloud.google.com/scheduler/docs/ 
```

Comment:
We should try to avoid raw URLs for readability.

```suggestion
Refer to the [Google Cloud Scheduler documentation](https://cloud.google.com/scheduler/docs/) for more information on the App Engine dependency.
```